### DISCUSSION TITLE
Welcome to BSManager Discussions!

### DISCUSSION THREAD

**Discussion by mann1x** [mann1x/BSManager#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Request features.
  * Look for help troubleshooting issues.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Majorkrabbe** [comment]
I am trying to get the custom sabers in my beat saber using BSmanager, but I can not get it to work. As I read on the internet there should be something called Saber factory. But it is not present in BSmanager. Could you please explain how to get custom sabers?
